tests/scripts/generate_test_keys.py

```diff
@@ -12,6 +12,7 @@
 import argparse
 import scripts_path # pylint: disable=unused-import
 from mbedtls_dev.asymmetric_key_data import ASYMMETRIC_KEY_DATA
+from mbedtls_dev.build_tree import guess_project_root
 
 OUTPUT_HEADER_FILE = os.path.dirname(os.path.abspath(__file__)) + "/../src/test_keys.h"
```

Comment:
I think we should remove this line, as it's superseded by `default_output_path` in `main()` (and also it's not been updated to the "right" way of handling paths).